test/e2e/apimachinery/etcd_failure.go

```diff
@@ -130,7 +131,7 @@ func checkExistingRCRecovers(f *framework.Framework) {
 	framework.ExpectNoError(wait.Poll(time.Millisecond*500, time.Second*60, func() (bool, error) {
 		options := metav1.ListOptions{LabelSelector: rcSelector.String()}
 		pods, err := podClient.List(options)
-		Expect(err).NotTo(HaveOccurred())
+		Expect(err).NotTo(HaveOccurred(), "failed to list pods in namespace: %s, that match label selector: %s", f.Namespace.Name, rcSelector.String())
```

Comment:
Overall this PR LGTM, but I'm curious if there is a way we could get the namespace name from `podClient` here? Something about having to refer back N>10 lines to see why/how to craft this message seems less than ideal